src/com/projectkorra/projectkorra/board/BendingBoardManager.java

```diff
@@ -34,26 +40,48 @@
 	private static boolean enabled;
 
 	public static void setup() {
-		initialize(true);
+		loadDisabledPlayers();
+		initialize();
 		Bukkit.getOnlinePlayers().forEach(BendingBoardManager::canUseScoreboard);
 	}
 
 	public static void reload() {
 		scoreboardPlayers.values().forEach(BendingBoardInstance::disableScoreboard);
 		scoreboardPlayers.clear();
-		initialize(false);
+		initialize();
 	}
 
-	private static void initialize(boolean initial) {
-		if (initial) loadDisabledPlayers();
+	private static void initialize() {
 		enabled = ConfigManager.getConfig().getBoolean("Properties.BendingBoard");
+		
 		disabledWorlds.clear();
 		disabledWorlds.addAll(ConfigManager.getConfig().getStringList("Properties.DisabledWorlds"));
+		
+		if (ConfigManager.languageConfig.get().contains("Boards.Extra")) {
```

Comment:
I think there's a typo here. It should be Board.Extras not Boards.Extra I think.